src/nvector/parallel/nvector_parallel.c

```diff
@@ -107,6 +107,7 @@ N_Vector N_VNewEmpty_Parallel(MPI_Comm comm,
   v->ops->nvsetarraypointer = N_VSetArrayPointer_Parallel;
   v->ops->nvgetcommunicator = N_VGetCommunicator_Parallel;
   v->ops->nvgetlength       = N_VGetLength_Parallel;
+  v->ops->nvgetlocallength  = N_VGetLocalLength_Parallel;
```

Comment:
The implementation of this function is missing

Reply:
Its there on line 277

Reply:
Yep, was looking for it in the diff but it already exists. All good.